pages/give/endowments/index.vue

```diff
@@ -2,14 +2,14 @@
     <!-- v-ifs working on section wrappers without v-show -->
     <main id="main" class="page page-endowment-listing">
         <masthead-secondary :title="page.title" :text="page.text" />
-        <search-generic
+        <!-- <search-generic
```

Comment:
we only want to remove the donor filter and keep the search, see services and resource listing page for no filters example